src/com/facebook/buck/android/AdbHelper.java

```diff
@@ -902,4 +956,23 @@ public static String tryToExtractPackageNameFromManifest(
       throw new HumanReadableException("Could not extract package name from %s", pathToManifest);
     }
   }
+
+  public static String tryToExtractInstrumentationTestRunnerFromManifest(
+      InstallableApk androidBinaryRule,
+      ExecutionContext context) {
+    Path pathToManifest = androidBinaryRule.getManifestPath();
```

Comment:
It should.  By the time the `androidBinaryRule` has run, all of its deps should have run, which means this `Path` should exist.